tests/unit/opera/compare/conftest.py

```diff
@@ -0,0 +1,269 @@
+import pytest
```

Comment:
Nice unit tests! Have you thought of adding an integration test as well? It'd be cool to have an integration test where opera diff would be used in action. You can add this test (after you rebase this branch onto master) to `tests/integration/cli_commands` where a testing script that tests all opera CLI commands resides, or you can create a fresh new integration test folder.